device-types/Cisco/Meraki-MS355-48X.yaml

```diff
@@ -0,0 +1,129 @@
+---
+manufacturer: Cisco
+model: Meraki MS355-48X
+slug: cisco-meraki-ms355-48x
+part_number: MS355-48X-HW
+u_height: 1
+is_full_depth: false
+comments: '[MS355 Overview and Specifications](https://documentation.meraki.com/MS/MS_Overview_and_Specifications/MS355_Overview_and_Specifications)'
+console-ports:
+  - name: Management
+    type: rj-45
+power-ports:
+  - name: Slot 1
+    type: iec-60320-c14
+  - name: Slot 2
+    type: iec-60320-c14
```

Comment:
Are these PSU's modular? If so, they should be modelled as `module-bays` on this device, and the associated PSU modules should be created in the module-types folder if they do not already exist